lib/spack/spack/ci.py

```diff
@@ -705,14 +722,51 @@ def generate_gitlab_ci_yaml(
             files (spack.yaml, spack.lock), etc should be written.  GitLab
             requires this to be within the project directory.
     """
+    rev1, rev2 = get_change_revisions()
+    tty.warn(f"Got following revisions: rev1={rev1}, rev2={rev2}")
+
+    # Get the joined "ci" config with all of the current scopes resolved
+    ci_config = cfg.get("ci")
+    spack_prune_untouched = os.environ.get("SPACK_PRUNE_UNTOUCHED", None)
```

Comment:
I feel like the diff should have a removal of this line from somewhere else, since `spack_prune_untouched` was referenced in the code below.  Am I just missing it?

Reply:
Ah yeah, I did a copy not a cut.